@navikt/ds-react/stepper/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@navikt/ds-react-stepper",
+  "author": "NAV Designsystem team",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/navikt/nav-frontend-moduler.git"
+  },
+  "dependencies": {
+    "@navikt/ds-css": "0.1.0-beta.6",
+    "classnames": "^2.2.6"
```

Comment:
Hvordan funker denne uten at `@navikt/ds-icons` er listet opp som en avhengighet?

Reply:
Antar det er siden yarn workspaces linker ds-icons i node_modules for prosjektet